scripts/publish.rs

```diff
@@ -0,0 +1,127 @@
+use std::env;
```

Comment:
Could we please add documentation at the top that explains the script purpose and how to run it. It's kinda unconventional way to execute the script, so people won't know it: 

```
rustc scripts/publish.rs --crate-type bin --out-dir scripts
```